packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/fileOutputReporter/FileOutputReporter.mdx

```diff
@@ -0,0 +1,23 @@
+# Wdio File Output Reporter
```

Comment:
Should this go inside the [Wdio Report](https://github.com/cerner/terra-toolkit/tree/fileoutput-reporter/packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/wdio-reporters.3) directory to group with [other reporters](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-functional-testing/wdio-reporters/spec-reporter)?

Reply:
addressed https://github.com/cerner/terra-toolkit/pull/665/files#diff-11886e1f7aa3ee1e87b367b174084d9692573d2bc64322691e8e0b6d0ef1a898